cmd/client/main.go

```diff
@@ -35,36 +38,26 @@ func mainWithArgs(ctx context.Context, args []string, logger golog.Logger) error
 }
 
 func runClient(ctx context.Context, deviceAddress string, logger golog.Logger) (err error) {
-	robotClient, err := client.NewClient(ctx, deviceAddress, logger)
-	if err != nil {
-		return err
-	}
-	defer func() {
-		err = multierr.Combine(err, robotClient.Close())
-	}()
-	names := robotClient.LidarNames()
-	if len(names) == 0 {
-		return errors.New("no lidar devices found")
-	}
-	lidarDevice := robotClient.LidarByName(names[0])
 
-	if err := lidarDevice.Start(ctx); err != nil {
+	robotClient, err := client.New(ctx, deviceAddress, logger, client.WithDialOptions(rpc.WithInsecure()))
+	if err != nil {
 		return err
 	}
 
 	defer func() {
-		err = multierr.Combine(err, lidarDevice.Stop(context.Background()))
+		err = multierr.Combine(err, robotClient.Close(ctx))
 	}()
 
-	info, err := lidarDevice.Info(ctx)
-	if err != nil {
-		return err
+	cameraDevice, ok := robotClient.CameraByName(robotClient.CameraNames()[0])
+	if !ok {
+		return fmt.Errorf("failed to find component")
 	}
-	logger.Infow("lidar", "info", info)
 
-	ticker := time.NewTicker(time.Second)
+	// Run loop
+	ticker := time.NewTicker(5 * time.Second)
```

Comment:
This is malleable, I chose 5s originally  to allow .las files not to be generated too quickly. My opinion is we stick with 5s for now until we need to speed it up (potentially for slam or for other applications)

Reply:
ok, will keep for now. 

Done.

Reply:
Actually, taking this back up. Should I use `selectContextOrWait` instead of the ticker here, @Otterverse ?

Reply:
Actually, yeah, probably so. Good catch!